python/ccxt/async_support/base/exchange.py

```diff
@@ -2,7 +2,7 @@
 
 # -----------------------------------------------------------------------------
 
-__version__ = '1.63.11'
+__version__ = '1.63.10'
```

Comment:
If possible, please drop the auto-generated files from this PR/branch.
One more thing: somehow bitfinex2.js ends up in every single PR.
It probably needs some git-rebasing.